### PR TITLE
Station vehicles WIP

### PR DESCRIPTION
**Pull request by JoanRisu**
This is a WIP PR

This is a new vehicle plus trolleys, both that I need some assistance in with coding. I also could use some ideas on what else can be added or what should be removed as well as sprite critiques.

Currently, I'm more concerned with getting layers to update depending on direction just like the chairs and the pussywagon. I'm also focusing on trying to get the mob offset correctly for each direction.

Note: Please do not merge this until #298 is merged.

### CODE REVIEW

**Comment by SpadesNeil** [comment]
Marking do not merge for now.

**Comment by SpadesNeil** [comment]
Also make sure, if you haven't already, they respond to EMPs.

**Comment by JoanRisu** [comment]
They should already based on the vehicle code but I can go in later and set it up.

File: code/modules/vehicles/Securitrain_vr.dm [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+	if(!lead && !tow)
+		anchored = 0
+	else
+		anchored = 1
+
+//-----------------------------------------------------
+//Update layer stuff
+//
+//This is supposed to update the layers and put the mob in the correct spot. 
+//Pls help squirrel get this to work. ;m;
```
Comment (JoanRisu) [inline code/modules/vehicles/Securitrain_vr.dm]: Dis stuff is the stuff I need help getting to work properly. ;m;

**Comment by Datraen** [comment]
For what it's worth, I poked my head over to see what you guys were up to, and I believe that more vehicles would be desired by the rest of the Polaris staff, as well as myself.

**Comment by SpadesNeil** [comment]
Sounds good! Once we get this done we can send these changes upstream, or you can nab them yourself for a PR. Whatever happens first.

**Comment by Datraen** [comment]
I'd have to ensure that it's not reliant on anything regarding any of your specific code, but we'll see how it goes.

**Comment by JoanRisu** [comment]
I really don't know what else I can do here. I'm not experienced enough to do the fine tuning this code needs. I really want someone else to help work on this.

**Comment by JoanRisu** [comment]
Should I close this and PR it into a WIP branch or something? :\

**Comment by JoanRisu** [comment]
Buuuummmmmmmmmmmmp. What doooooo

**Pull request closed without merging**